src/overlays/actors/ovl_En_Diving_Game/z_en_diving_game.c

```diff
@@ -80,7 +80,7 @@ static ColliderCylinderInit sCylinderInit = {
         ELEM_MATERIAL_UNK0,
         { 0x00000000, 0x00, 0x00 },
         { 0x00000000, 0x00, 0x00 },
-        ATELEM_NONE | ATELEM_SFX_NORMAL,
+        ATELEM_NONE,
```

Comment:
what is `ATELEM_SFX_NORMAL`, and why is it removed from here but added elsewhere?

Reply:
per collision_check.h that's `Hit sound effect based on AC collider's type`
this flag may only be used when the collider is AT, so if the collider is not AT (ATELEM_NONE) then it's irrelevant to set it (`ATELEM_SFX_NORMAL` is 0)